.vscode/tasks.json

```diff
@@ -0,0 +1,20 @@
+{
+    // See https://go.microsoft.com/fwlink/?LinkId=733558
+    // for the documentation about the tasks.json format
+    "version": "2.0.0",
+    "tasks": [
+        {
+            "label": "Build OPT",
+            "type": "shell",
+            "command": "./tools/scripts/buildModOPT.bat both",
```

Comment:
hier dann einfach noch das `noPause` nach dem `both` als Argument mit angeben, dann funktioniert das auch noch, wenn du die entsprechenden Zeilen in den Skripten wieder entkommentierst :)

Reply:
Den Param hab ich wohl übersehen, ja :) .